src/app/components/Header/index.tsx

```diff
@@ -444,13 +456,18 @@ export function Header() {
                   <FontAwesomeIcon icon={faChevronDown} size="xs" />
                 </div>
               </NavPopover> */}
+              </div>
             </div>
           </div>
+
           <div className="tw-flex tw-justify-start tw-items-center">
             <Link
-              className={classNames('tw-btn-action xl:tw-mr-2', {
-                'tw-hidden': !connected,
-              })}
+              className={classNames(
+                'tw-btn-action tw-hidden sm:tw-flex tw-mr-2 xl:tw-mr-4',
```

Comment:
Add margin on the left, switch to mr-4 a breakpoint later. to cleanup issues around 1200px 
```suggestion
                'tw-btn-action tw-hidden sm:tw-flex tw-mx-2 2xl:tw-mr-4',
```